cluster/plan.go

```diff
@@ -167,6 +195,37 @@ func osLimitationFilter(osType string, processes map[string]v3.Process) map[stri
 	return processes
 }
 
+func osLimitationFilterCluster(osType string, processes map[string]v3.RKEProcess) map[string]v3.RKEProcess {
+	if osType != "windows" {
+		return processes
+	}
+
+	// windows limitations
+	for name, process := range processes {
+		// doesn't support host network on windows
+		if process.NetworkMode == "host" {
+			process.NetworkMode = ""
+		}
+
+		// doesn't support PID on windows
+		if process.PidMode != "" {
```

Comment:
nit: you can directly set `process.PidMode` to `""` without checking and same for `process.Privileged`